src/main/java/com/projectg/geyserupdater/common/pojo/Bungeecord.java

```diff
@@ -0,0 +1,6 @@
+package com.projectg.geyserupdater.common.pojo;
+
+public class Bungeecord {
+    public String name;
+    public String sha256;
+}
```

Comment:
1) Can we have a better package name than `pojo`
2) Seems like each platform pojo is the exact same? Considering the downloads api will only have breaking changes under a newer version, can we just have one pojo class for all platforms?
3) newline at end of file for all new files please, i'm sorry